packages/components/Callout/macos/CalloutView.swift

```diff
@@ -224,16 +399,12 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 	}
 
 	/// Calculates the NSRect of the anchorView in the coordinate space of the current screen
-	private func calculateAnchorViewScreenRect() -> NSRect {
-		guard let anchorView = anchorView else {
-			preconditionFailure("No anchor view provided to position the Callout")
-		}
-
+	private func calculateAnchorViewScreenRect(anchorView: NSView, anchorBounds: NSRect) -> NSRect {
```

Comment:
It's not immediately obvious why you'd want `anchorBounds` as a separate argument instead of just using `anchorView.bounds`. You should add a comment explaining why.

Reply:
sought to clarify with a better variable name, per your recommendation of "Swiftier" function signatures :)